WcaOnRails/app/webpacker/components/RolesTab/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Button } from 'semantic-ui-react';
+import useLoadedData from '../../lib/hooks/useLoadedData';
+import { roleListUrl } from '../../lib/requests/routes.js.erb';
+import Errored from '../Requests/Errored';
+
+export default function RolesTab() {
+  const userId = window.location.pathname.split('/')[2];
+  const { data, loading, error } = useLoadedData(roleListUrl(userId));
+
+  if (loading) return 'Loading...';
```

Comment:
Is there something like a "Loading" component? Look at how other complex React components in our code do this